cmds/readlink/readlink.go

```diff
@@ -53,6 +53,11 @@ func readLink(file string) error {
 func main() {
 	var exitStatus int
 
+	if flag.NArg() == 0 {
```

Comment:
same here. I'd rather, if they don't supply args, just return no error and no output. That's a bit more unix style.